platform-db-runtime/src/main/java/com/softicar/platform/db/runtime/table/listener/IDbTableRowNotificationSet.java

```diff
@@ -14,36 +14,36 @@
  */
 public interface IDbTableRowNotificationSet<R> {
 
-	Collection<DbTableRowNotification<R>> getAllNotifications();
+	List<DbTableRowNotification<R>> getAllNotifications();
 
-	default Set<R> getAllTableRows() {
+	default List<R> getAllTableRows() {
 
 		return getAllNotifications()//
 			.stream()
 			.map(DbTableRowNotification::getTableRow)
-			.collect(Collectors.toCollection(HashSet::new));
+			.collect(Collectors.toCollection(HashList::new));
```

Comment:
here again, we remove the de-duplication when going to `List`

Reply:
not correct; see above